feature1/mais-um-arquivo.txt

```diff
@@ -0,0 +1,3 @@
+Esse arquivo também foi adicionado na branch feature1.
```

Comment:
It's really good that you have decided to add this line of code.

Reply:
Obrigado pelo comentário, Café.